go.sum

```diff
@@ -147,6 +147,8 @@ github.com/cockroachdb/logtags v0.0.0-20190617123548-eb05cc24525f h1:o/kfcElHqOi
 github.com/cockroachdb/logtags v0.0.0-20190617123548-eb05cc24525f/go.mod h1:i/u985jwjWRlyHXQbwatDASoW0RMlZ/3i9yJHE2xLkI=
 github.com/cockroachdb/pebble v0.0.0-20200618174744-bc7bb07c0a28 h1:OGzpqbaARCNI/VvrKMydw22zClKW3FX2fKDanLlbbXA=
 github.com/cockroachdb/pebble v0.0.0-20200618174744-bc7bb07c0a28/go.mod h1:crLnbSFbwAcQNs9FPfI1avHb5BqVgqZcr4r+IzpJ5FM=
```

Comment:
i think you also need to run a `go mod tidy` to delete the old items in this file.